easybuild/easyconfigs/r/RNA-SeQC/RNA-SeQC-1.1.8-foss-2016b-Java-1.8.0_121.eb

```diff
@@ -0,0 +1,37 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+
+easyblock = 'JAR'
+
+name = 'RNA-SeQC'
+version = '1.1.8'
+
+homepage = 'http://archive.broadinstitute.org/cancer/cga/%(namelower)s'
+description = """RNA-SeQC is a java program which computes a series of quality control metrics for
+ RNA-seq data. The input can be one or more BAM files. The output consists of HTML reports and tab
+ delimited files of metrics data. This program can be valuable for comparing sequencing quality
+ across different samples or experiments to evaluate different experimental parameters. It can
+ also be run on individual samples as a means of quality control before continuing with downstream
+ analysis."""
+
+toolchain = {'name': 'foss', 'version': '2016b'}
+
+source_urls = ['https://data.broadinstitute.org/cancer/cga/tools/rnaseqc/']
+sources = ['%(name)s_v%(version)s.jar']
+
+java = 'Java'
+javaver = '1.8.0_121'
+versionsuffix = '-%s-%s' % (java, javaver)
+dependencies = [(java, javaver)]
```

Comment:
Please use the `%(javaver)s` template (and move up the `versionsuffix = ...` line below `version = ...`

```python
version = '1.1.8'
versionsuffix = '-Java-%(javaver)s'
...
dependencies = [('Java', '1.8.0_121')]
```

Reply:
@boegel I changed this.